docs/changelog.txt

```diff
@@ -1,3 +1,54 @@
+
+- Feature: Industries now support up to 16 accepted cargos and 16 produced cargos
+- Feature: Houses now support up to 16 accepted cargos
+- Feature: Support for RoadTypes (a.k.a NRT)
+- Feature: Allow use of switches and random switches as procedures, reducing repetition of switches
+- Feature: Increase railtype ID range from 16 to 64, to support 64 railtypes
+- Feature: Railtype flags RAILTYPE_FLAG_ALLOW_90DEG and RAILTYPE_FLAG_DISALLOW_90DEG for 90 degree curves
+- Feature: Railtype flags RAILTYPE_FLAG_HIDDEN and RAILTYPE_FLAG_PRECOMBINED for precombined spriteset type
+- Feature: Add ANIM_TRIGGER_APT_AIRPLANE_LANDS (follows OpenTTD #7182)
+- Feature: Add station var nearby_tile_grfid (0x6A)
+- Feature: PUSH/POP_COLOUR codes for strings
+- Feature: Support industry vars production_rate (0x70) and transported_last_month_pct (0x71)
+- Feature: Debug_print for GRF parameters
+- Feature: nmlc option to enable debug mode for parser
+- Feature: nmlc option to force regeneration of parser tables
+- Feature: Add TTDPatchFlag 256_persistent_registers (0x80)
+- Change: improved colouring of nmlc errors and warnings (where shell/terminal supports this)
+- Change: Make sprite encoder warnings more informative (#89)
+- Fix #36: Global variable traffic_side was inconsistent with global parameter traffic_side
+- Fix #39: Add compatibility with >=pillow-7.0.0
+- Fix #58: Loosen Pillow version constraint
+- Fix #52: time.clock() has been removed in python 3.8
+- Fix #57: Check coherency of GRF parameters limits
+- Fix 014febc: Wrong link to LICENSE file
+- Fix 78cd84c: Restore 'setup.py install' usability on windows
+- Fix: Add missing parameter in industry and airport tiles in RandomAction2
+- Fix: Convert some code flows to context managers (eliminates 'too many open files' error on macOS with larger grfs)
+- Fix: LOAD_TEMP() 'value' attribute is empty
+- Fix: Update various sprite counts
+- Codechange: Visual Studio Editor Script
+- Codechange: Install_requires in setup.py
+- Codechange: pyinstaller spec file
+- Codechange: Allow PLY to generate parsing/lexing tables (performance improvement)
+- Codechange: Avoid duplicating rail/road/tramtype table code
+- Codechange: Update nml version detection to support Git using similar approach to OpenTTD
+- Codechange: Mechanical cleanups of Python code
+- Codechange: Remove cx_Freeze
+- Codechange: Rewrite setup.py
+- Codechange: Simplify Pillow imports
+- Codechange: Simplify Pillow version detection
+- Codechange: Move license file to root folder
+- Doc: Update README
+- Doc: 0.3 changed 'shorten_vehicle' to 'length' - this wasn't updated in one place in train example nml
+- Doc: Add example train wagon showing sprite stack use
+- Doc: Fix 404 on pillow install instructions link
+- Doc/Fix: Update the stated version requirements
+- Cleanup: Industry vars production_rate_1 (0x8E) and production_rate_2 (0x8F) are deprecated and replaced by new parametric var production_rate (0x70)
```

Comment:
Do the old ones still work? If not this should be much higher.

Reply:
Indeed, if it's not an actual removal, it's not a "cleanup", rather a "change"

Reply:
+1 this will cause some people's compiles to fail;  I really think this has to be out-of-order and follow directly from the 'Feature: production_rate 0x70 blah blah' line.